packages/common/src/initiatives/HubInitiative.ts

```diff
@@ -16,7 +16,7 @@ import {
   updateInitiative,
 } from "./HubInitiatives";
 
-import { Catalog } from "../search/Catalog";
+import { Catalog } from "../search";
```

Comment:
I can' t for the life of me figure out why this works and why `../search/Catalog` doesn't.

I wonder if you removed the re-export from index.ts if `../search/Catalog` would have worked. If so, one idea for avoiding this going forward would be to remove the  index.ts files that only re-export. This would also help avoid circular dependencies caused by `import`ing from `../index` or `../.` or any of those variants. Before we consider something like that, should think through the folder structure and what [package entrypoints](https://nodejs.org/api/packages.html#package-entry-points) we want to support.

Until then, my only suggestion is to add comments to this line and the others like it to indicate that it _has_ to be done this way otherwise we will break node.